examples/hello/Hello/extracted/Hello.fs

```diff
@@ -0,0 +1,10 @@
+#light "off"
+module Hello
+open Prims
+open FStar_Pervasives
```

Comment:
@kant2002 I think these extracted files were added by mistake?

Reply:
Yes. I already remove them and start ignoring.